src/qcommon/files.cpp

```diff
@@ -3281,6 +3327,10 @@ static void FS_Startup( const char *gameName ) {
 	assetsPath = Sys_DefaultAssetsPath();
 	fs_assetspath = Cvar_Get("fs_assetspath", assetsPath ? assetsPath : "", CVAR_INIT | CVAR_VM_NOWRITE);
 
+	assetsPathJKA = Sys_DefaultAssetsPathJKA();
+	fs_assetspathJKA = Cvar_Get("fs_assetspathJKA", assetsPathJKA ? assetsPathJKA : "", CVAR_INIT | CVAR_VM_NOWRITE);
+	fs_noJKA = Cvar_Get("fs_noJKA", "0", CVAR_ARCHIVE);
```

Comment:
Since we have an `fs_restart` command for in-place reload before a `vid_restart` I don't think we need it for this cvar.

Reply:
CVAR_LATCH cvar gets set next time Cvar_Get() is called on that cvar, so not necessarily vid_restart. in this case fs_restart will call FS_Startup() with `Cvar_Get("fs_assetspathJKA" ...` and it will become "unlatched"

Reply:
Added CVAR_LATCH flag as of 775cd2fecb73a59843931577bc1960b8d2abcad4.